src/main/java/org/sonarlint/intellij/actions/filters/SecurityHotspotFilters.java

```diff
@@ -0,0 +1,19 @@
+package org.sonarlint.intellij.actions.filters;
+
+public enum SecurityHotspotFilters {
+
+  SHOW_ALL("Show all"),
+  LOCAL_ONLY("Local only"),
+  MATCHED_ON_SONARQUBE("Matched on SonarQube");
```

Comment:
Matched with SQ instead of on ? Not sure

Reply:
I've changed to `Existing on SonarQube` as we discussed on MMF specification